### ISSUE TITLE
Upgrading google-api-client version to 0.9.x

### ISSUE THREAD

**Issue by umate** [gimite/google-drive-ruby#190]
Hello,

Any chances that library will be supporting google-api-client ~> 0.9.0 ?

**Comment by gimite** [comment]
Yeah I want to do it eventually, but I'm a bit wondering whether to do it now because it will cause backward incompatibility for the API of this library. google-api-client 0.9 is incompatible with 0.8, so it's tricky to support both.

What is your motivation to use google-api-client 0.9?

**Comment by treby** [comment]
I have same wondering with @umate.

We use this library for our old system using google-api-ruby-client 0.8. But recently, we're migrating our system and the new one is using google-api-ruby-client 0.9.
Because there are many features using google-api-ruby-client, it's not a good idea to use 0.8 only for this library.

IMHO, like google-api-ruby-client, it seems a good idea providing support 0.8 and 0.9 with different versions. Of cause, we still have some problems of backport costs...
What do you think? @gimite

**Comment by gimite** [comment]
Yeah, that sounds like a valid reason. OK, I will work on 0.9 support when I have time.

**Comment by paulz** [comment]
That would help us too, on my project we can't install this gem:
```
Bundler could not find compatible versions for gem "google-api-client":
  In snapshot (Gemfile.lock):
    google-api-client (= 0.8.6)

  In Gemfile:
    google_drive was resolved to 1.0.6, which depends on
      google-api-client (< 0.9, >= 0.7.0)

    fastlane (~> 1.66) was resolved to 1.66.0, which depends on
      supply (< 1.0.0, >= 0.4.0) was resolved to 0.4.0, which depends on
        google-api-client (~> 0.9.1)
```
Thank you for the awesome gem!

**Comment by paulz** [comment]
We had to migrate to use command line utility for Google Drive:
https://github.com/prasmussen/gdrive

since this gem was causing conflicts due to outdated google-api-client requirements

**Comment by gimite** [comment]
I have pushed google_drive 2.0.0.pre1 which uses google-api-client 0.9.x.

Try it with:

    $ gem install google_drive --pre

**Comment by paulz** [comment]
thank you @gimite 
we had to go with
```
brew install gdrive
```

**Comment by gimite** [comment]
Released google_drive 2.0.0 with the change.
